test/PerformanceTests/SerializationComparisonsTests/ExperimentTests/WriterTests.cs

```diff
@@ -0,0 +1,82 @@
+//---------------------------------------------------------------------
+// <copyright file="WriterTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ExperimentsLib;
+using Xunit;
+
+namespace ExperimentTests
+{
+    public class WriterTests
+    {
+        public static IEnumerable<Customer> data = CustomerDataSet.GetCustomers(4);
+        public static WriterCollection<IEnumerable<Customer>> writers = DefaultWriterCollection.Create();
+        public static IEnumerable<object[]> WriterNames { get; } = writers.GetWriterNames()
+            .Where(n => !n.Contains("NoOp"))
+            .Select(n => new string[] { n });
+
+        public static IEnumerable<object[]> noOpWriterNames() =>

```

Comment:
This is public so maybe we should use PascalCase...

Reply:
This should be public cause it's used by `[MemberData]`. I will rename it.